hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -242,6 +242,9 @@ public enum Units { TB, GB, MB, KB, B }
    */
   public static final String OZONE_SCM_DATANODE_ID_FILE_DEFAULT = "datanode.id";
 
+  public static final String OZONE_SCM_DATANODE_DISK_BALANCER_INFO_DEFAULT =
+      "diskBalancer.info";
```

Comment:
OZONE_SCM_DATANODE_DISK_BALANCER_INFO_DEFAULT -> OZONE_SCM_DATANODE_DISK_BALANCER_INFO_FILE_DEFAULT

Reply:
@symious , this is not addressed yet.